library/alloc/Cargo.toml

```diff
@@ -38,3 +38,5 @@ compiler-builtins-mangled-names = ["compiler_builtins/mangled-names"]
 compiler-builtins-weak-intrinsics = ["compiler_builtins/weak-intrinsics"]
 # Make panics and failed asserts immediately abort without formatting any message
 panic_immediate_abort = []
+# Choose algorithms that are optimized for binary size instead of runtime performance
+optimize_for_size = []
```

Comment:
It's not clear to me why this isn't forwarding the feature to `core`. `panic_immediate_abort` should probably be doing it too.

Reply:
I copied the `panic_immediate_abort` flag so that's why the new one didn't have it either.
It makes sense to forward. I added the forwarding to both.